slotfills/sections/checkbox.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { PanelRow, CheckboxControl } from '@wordpress/components';
+import FMObject from '@/interfaces/fm-object';
 import Field from '@/interfaces/field';
 
 interface CheckboxProps {
   field: Field;
   index?: number;
   label?: string;
-  valueHook: (key: number | string) => [any | any[], Function];
+  valueHook: (key: number | string) =>
+  [boolean | string | FMObject | boolean[] | string[] | FMObject[], Function];
```

Comment:
That's a large amount of available types! Would it be reasonable just to supply `any`? Or are these values guaranteed? Is there anything missing here?

Reply:
@stevenslack the goal here was to move away from `any`. I think everything is guaranteed and thought of, but I'm not 100% sure. `valueHook` will be `usePostMetaValue` or a function that pulls an indexed value from an array.